app/src/main/java/com/folen/androidshowreel/activity/MainActivity.java

```diff
@@ -26,11 +26,10 @@
 
 public class MainActivity extends BaseActivity {
 
-    RecyclerView recyclerView;
+    private RecyclerView recyclerView;
```

Comment:
Te zmiany mialy byc w innym pull request, ale juz to olejmy. Nie sa one zwiazane z tym co opisales w nazwie PR